allennlp/tests/models/sniff_test.py

```diff
@@ -112,7 +112,7 @@ def test_constituency_parsing(self):
         result = predictor.predict_json({"sentence": sentence})
 
         assert result["tokens"] == ["Pierre", "Vinken", "died", "aged", "81", ";", "immortalised", "aged", "61", "."]
-        assert result["trees"] == "(S (NP (NNP Pierre) (NNP Vinken)) (VP (VP (VBD died) (NP (JJ aged) (CD 81))) (, ;) (VP (VBD immortalised) (S (ADJP (VBN aged) (NP (CD 61)))))) (. .))"
+        assert result["trees"] == "(S (S (NP (NNP Pierre) (NNP Vinken)) (VP (VBD died) (NP (JJ aged) (CD 81)))) (: ;) (S (VP (VBN immortalised) (S (ADJP (VBN aged) (NP (CD 61)))))) (. .))"
```

Comment:
Looking at this a little more closely, the new parse is more correct, especially the VBN instead of VBD for "immortalised" (which may have been the reason the parse switched).  So, I don't think this is a regression.